src/examples/footer/compact.html

```diff
@@ -0,0 +1,17 @@
+<footer class="flex flex-col sm:flex-row gap-8 justify-between p-10 bg-base-200">
+
+    <!-- Brand -->
+    <aside>
+        <span class="fa-brands fa-superpowers mr-2 text-4xl align-center text-primary"></span>
+        <p class="text-2xl">Company</p>
+        <small>Copyright © 2024 - All rights reserved</small>
+    </aside>
+
+    <!-- Socials -->
+    <nav class="flex gap-4">
+        <a class="fa-brands fa-github text-2xl"></a>
+        <a class="fa-brands fa-twitter text-2xl"></a>
+        <a class="fa-brands fa-facebook text-2xl"></a>
+        <a class="fa-brands fa-youtube text-2xl"></a>
```

Comment:
Add `hover:cursor-pointer` or `btn` so users can see that these icons are clickable